cmd/scw/testdata/test-all-usage-lb-acl-delete-usage.golden

```diff
@@ -0,0 +1,19 @@
+🎲🎲🎲 EXIT CODE: 0 🎲🎲🎲
+🟥🟥🟥 STDERR️️ 🟥🟥🟥️
+Delete an ACL.
+
+USAGE:
+  scw lb acl delete [arg=value ...]
+
+ARGS:
+  aclid             ID of your ACL ressource
```

Comment:
This one is weird, should be `acl-id` 🤔

Reply:
This map needs to be updated in order to fix it https://github.com/scaleway/scaleway-sdk-go/blob/master/strcase/bash_arg.go#L5

Reply:
https://github.com/scaleway/scaleway-sdk-go/pull/488

Reply:
@jerome-quere What do you think about it? It will create a small breaking change.

Reply:
This PR was merged